src/analysis/special_functions/pow.lean

```diff
@@ -1272,6 +1272,12 @@ lemma monotone_rpow_of_nonneg {z : ℝ} (h : 0 ≤ z) : monotone (λ x : ℝ≥0
 h.eq_or_lt.elim (λ h0, h0 ▸ by simp only [rpow_zero, monotone_const])
   (λ h0, (strict_mono_rpow_of_pos h0).monotone)
 
+/-- Bundles `λ x : ℝ≥0∞, x ^ y` into an order isomorphism when `y : ℝ` is positive,
+where the inverse is `λ x : ℝ≥0∞, x ^ (1 / y)`. -/
+@[simps] def order_iso_rpow {y : ℝ} (hy : 0 < y) : ℝ≥0∞ ≃o ℝ≥0∞ :=
```

Comment:
`@[simps]` generates bad lemma for `symm_apply`. Could you please use 
```suggestion
@[simps apply] def order_iso_rpow (y : ℝ) (hy : 0 < y) : ℝ≥0∞ ≃o ℝ≥0∞ :=
```
here and add a lemma that states `(order_iso_rpow y hy).symm = order_iso_rpow y⁻¹ (inv_pos.2 hy)` (or with `1 / y` instead of `y⁻¹`)?

Reply:
done